forms-flow-web/src/services/UserService.js

```diff
@@ -49,11 +48,12 @@ const initKeycloak = (store, ...rest) => {
     checkLoginIframe: false,
   }).then((authenticated) => {
     if (authenticated) {
-      if (KeycloakData.resourceAccess[clientId]) {
-        const UserRoles = KeycloakData.resourceAccess[clientId].roles;
+      const tokenParsed = KeycloakData.tokenParsed || {};
+      const UserRoles = tokenParsed.roles || tokenParsed.role || tokenParsed.client_roles || [];
```

Comment:
i think this file is no longer used in our application, and checking UserRoles.lenght cause some login issue, that part we fixed in micro front end , so we need to bring the changes in micro front end  also,

Reply:
could you please update actions/cache: in bpmn ci